doc/installation.md

```diff
@@ -28,7 +28,7 @@ $ chmod a+x kerl
 $ mv kerl /usr/bin
 $ kerl list releases
 $ kerl build 18.2 r18.2 # this takes a while
-$ kerl install r18.2 /usr/lib/erlang
+$ kerl install r18.2 /usr/local/erlang
```

Comment:
In this case a system administrator must know not to install OS packaged Erlang, and not to install any OS package having Erlang in dependencies. This actions are highly unprobable though, but eventually could happen. However if we use our own Erlang this couldn't happen even in theory, and both ways are equally quick. Just my 2 cents.